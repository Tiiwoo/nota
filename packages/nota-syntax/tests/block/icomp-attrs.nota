Hello @em[id: "ex"]{world}