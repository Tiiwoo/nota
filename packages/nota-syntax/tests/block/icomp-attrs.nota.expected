el(
  "p",
  {},
  "Hello ",
  el(
    "em",
    {
      id: "ex",
    },
    "world"
  )
);